docs/samples/Microsoft.ML.Samples/Dynamic/ModelOperations/SaveLoadModelFile.cs

```diff
@@ -25,7 +25,8 @@ public static void Example()
             var outputColumnName = nameof(Transformation.Key);
 
             // Transform.
-            ITransformer model = mlContext.Transforms.Conversion.MapValueToKey(outputColumnName, inputColumnName).Fit(dataView);
+            ITransformer model = mlContext.Transforms.Conversion.MapValueToKey(
+                outputColumnName, inputColumnName).Fit(dataView);
```

Comment:
For cases like this, I would rather we break per method call, than to break between a method and its parameter list.

For example:

```C#
    // If you have chained method calls, line-break each method call
    Enumerable.Range(0, numRows)
        .Select(i => i.ToString())
        .ToArray();
```

That is much more readable than:

```C#

object o = foo.methodCall(
    methodCallArg1, methodCallArg2).OtherMethodCall(arg3)
```